bundles/SyliusAddressingBundle/installation.rst

```diff
@@ -35,7 +35,7 @@ Don't worry, everything was automatically installed via Composer.
     {
         $bundles = array(
             new FOS\RestBundle\FOSRestBundle(),
-            new JMS\SerializerBundle\JMSSerializerBundle($this),
+            new JMS\SerializerBundle\JMSSerializerBundle($this), # Optionnal, Only for API purpose
```

Comment:
it doesn't matter much, but it's more natural to say something like:

// Only Required if providing API access to your content

(or any other word than content)

In any case, Optional is spelled wrong, and you should probably use a // for the comment and not a #